qemu/tests/cfg/numa_opts.cfg

```diff
@@ -8,6 +8,7 @@
   kill_vm = yes
   kill_vm_gracefully = no
   encode_video_files = no
+  pre_command = "sync && echo 3 >/proc/sys/vm/drop_caches"
```

Comment:
Better append `;` at the end the pre_command.

Reply:
Thanks, updated.